src/os/unix/fs.rs

```diff
@@ -40,14 +40,25 @@ cfg_if! {
             /// `0o777`.
             fn mode(&mut self, mode: u32) -> &mut Self;
         }
+    } else {
+        #[doc(inline)]
+        pub use std::os::unix::fs::DirBuilderExt;
+    }
+}
 
+cfg_if! {
+    if #[cfg(feature = "docs")] {
         /// Unix-specific extension methods for `DirEntry`.
         pub trait DirEntryExt {
             /// Returns the underlying `d_ino` field in the contained `dirent`
             /// structure.
             fn ino(&self) -> u64;
         }
+    }
+}
 
+cfg_if! {
```

Comment:
Let's also delete this section.

Reply:
If I delete these doc shims in `src/os/unix/fs.rs`, then I start to get errors like this:

```
Found invalid urls in async-std/target/doc/async_std/os/unix/fs/index.html:
        Linked file at path async-std/target/doc/std/fs/struct.OpenOptions.html does not exist!
```

I tried to trace back to where all of these are being referenced but wasn't successful. Do you still want to try and remove them?